packages/truffle-4/package.json

```diff
@@ -13,7 +13,6 @@
   "author": "Aragon Association <legal@aragon.org>",
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "truffle-hdwallet-provider": "0.0.3",
-    "truffle-hdwallet-provider-privkey": "0.3.0"
+    "truffle-hdwallet-provider": "0.0.3"
```

Comment:
Do you know if `@truffle/hdwallet-provider` is compatible with truffle@4? Could we use that version instead?

Reply:
I double check and is working as expected. I'll update to that version